schema.graphql

```diff
@@ -260,3 +260,11 @@ type ReserveMetrics @entity {
   totalFeeMinted: BigInt!
   allocations: [ReserveAllocationMetrics] @derivedFrom(field: "reserveMetrics")
 }
+
+type ReserveBalance @entity {
+  id: ID!
+  blockHeight: BigInt!
+  address: String!
```

Comment:
@toliaqat I am not sure about it. But what if the `blockHeight` is much larger than what a `BigInt` type can handle? Should we also use a `String` type for `blockHeight`?

Reply:
BigInt can store super long numbers such as: 10 ^(10^8) which i dont think we can physically reach

Reply:
BigInt is good enough.